### DISCUSSION TITLE
Don't merge stars and lists

### DISCUSSION THREAD

**Discussion by srgykuz** [community/community#8702] [Other Feature Feedback, Questions, & Ideas]
I understand your attention: to merge stars and lists so it can became one full feature. But please don't be in such a hurry! 

Literally, I waked up today, opened my GitHub profile, automatically move my mouse to the old place of "Stars", but it wasn't here:
![Screenshot from 2021-12-11 11-52-12](https://user-images.githubusercontent.com/20613502/145670639-a075ac2d-6756-4441-b021-a9e4587d5575.png)
And it moved here, to the very back and inconspicuous tab:
![Screenshot from 2021-12-11 11-53-11](https://user-images.githubusercontent.com/20613502/145670679-17a63b70-ee8d-48a2-94b3-b741d8963d97.png)

I understand you want to attract attention to the new feature. I understand you want to somehow improve "Stars" feature. But please don't overload what was already working fine.

"Stars" is one of most important feature for me. The repositories that I've put a star are really important for me: at any moment I can quickly find them for reference, example, idea, inspiration and anything else. "Stars" were placed near another important information - profile info. I used the top tab very rarely because all important information for me was placed at one place. This place was compact, easy to find and click, and elegant.

And now it is placed in the top bar, at the very last tab. Well, I can accept that it is placed near "Repositories" tab. At least it is looks logical: your repositories and your starred repositories. But why it have to be at very last position? Do you really think that "Projects" and "Packages" are being using more often than "Stars"? I don't use them at all! And now my most used feature is placed at the most inconvenient place.

Next, when I click on "Stars" and want to see at my stars, this is that I get:
![Screenshot from 2021-12-11 12-06-08](https://user-images.githubusercontent.com/20613502/145671028-b48f2e1f-f10a-49eb-ac86-522aa623772c.png)

I only see one repo. Seriously? I intended to see big list of recent repos, but instead got an empty space on half of screen. Now I need to scroll down, which is really so slow. Earlier behavior was super fast: quick click, big list of recent repos.

I understand purpose of lists - better organization of your stars. But for me "Stars" feature is not a feature that need such powerful organization tool like lists. I know my stars, I know how to quickly find it, I know what every project in this entire "Stars" list is important by itself, it doesn't need separate list. By the way, you already have a great mechanism to quickly find what you need - search form:
![Screenshot from 2021-12-11 12-09-31](https://user-images.githubusercontent.com/20613502/145671221-9b92dc96-4e0e-48d1-9548-7a09d817bf79.png)

Next, because you need to add ability to somehow manage lists, you added this tab:
![Screenshot from 2021-12-11 12-13-56](https://user-images.githubusercontent.com/20613502/145671256-6e9eaa08-07d9-421d-8001-c7bbfd6dc149.png)

But what the point to see this tab near every project?
![Screenshot from 2021-12-11 12-14-15](https://user-images.githubusercontent.com/20613502/145671277-2f92e72c-c13b-43d2-ba05-91d1d0284961.png)

I know this project have my star because it is in my "Stars" list! Why I have to look at this garbage information every time. I called this "garbage" because it really doesn't makes any sense - "starred" repo in your "Stars" list. I know that I can click on this button and repo will be unstarred. But I don't need a text for all of that. I understand purpose of like and unlike icon because it is "Stars" list, not abstract list.

So, let me summarize my grumbling:
1. I don't mind "Lists" feature. It is excellent feature. But don't merge it with "Stars" feature. If I ever had a list, I want it to consists of many repos, not only starred ones.
2. Don't take far away "Stars" feature. It is really important feature.

This summarizes all my thoughts. For me it is looks much more better in any aspect.
![Screenshot from 2021-12-11 12-24-40](https://user-images.githubusercontent.com/20613502/145671566-bf2acef9-18ef-4704-8bc3-4900d8f45038.png)

**Comment by fitojb** [comment]
GitHub is pulling a Twitter in this. I remember when they hijacked the Favorites feature in order to copy Facebook’s “Likes”.

**Reply by srgykuz** [reply]
Nothing extraordinary about copying. Just do it in logical manner. Current "Lists" feels like a sub-feature of "Stars" feature, which shouldn't be true logically.

**Comment by jsoref** [comment]
Fwiw, from an implementation detail perspective, this relates to the expected results in #8663

**Comment by AndrewRathbun** [comment]
Can anyone tell me how you view who has starred, forked, or watched a given repo like we were able to before? It was so nice to be able to do that in one click before but now I'm not seeing how you can do it a) at all and b) without diving into some menu somewhere which is already way too many clicks compared to how it was before. What am I missing here?

**Reply by LouisCAD** [reply]
It's now shown on the right pane/aside, took me a while to figure it out as well.

**Reply by AndrewRathbun** [reply]
> It's now shown on the right pane/aside, took me a while to figure it out as well.

![image](https://user-images.githubusercontent.com/36825567/146773175-b24611fb-3b05-4a62-8407-fdf4b729ba93.png)

Ahh here, okay thank you.

**Reply by srgykuz** [reply]
+1. It became very inconvenient
